src/forms/FormFieldImageUpload/FormFieldImageUpload.styled.ts

```diff
@@ -13,8 +13,8 @@ import CircularProgress from "@mui/material/CircularProgress";
 
 export const DragAndDropContainer = styled.div`
   align-items: center;
-  border: ${(pr) => (pr.isOver ? `1px dashed ${theme.colors.teal}` : "")};
-  background-color: ${(pr) => (pr.isOver ? "#008DA81A" : theme.colors.gray200)};
+  border: ${(pr) => (pr.isOver ? `1px dashed ${theme.newColors.realTeal["100"]}` : "")};
+  background-color: ${(pr) => (pr.isOver ? theme.newColors.realTeal["10"] : theme.colors.gray200)};

```

Comment:
Do we have the realTeal with 10% opacity? I think the lowest was 20%. If we don't then please change it to 20

Reply:
I just saw that you added the 0.1 to the themes file, but let's keep it in 20 because the figma file only uses multiples of 2